docs/pages/docs/guide/advanced/twoslash-support.mdx

```diff
@@ -22,6 +22,10 @@ todo.title = 'Hello'
 
 Number.parseInt('123', 10)
 //      ^|
+               // Just comments, so Popup will be
+               // not behind the viewport of `<code>`
+               // element due `position: absolute` style
```

Comment:
```suggestion
               // element due his `position: absolute` style
```

Reply:
`his` right?

Reply:
I did reference `his` for `Popup`, did I do mistake?

Reply:
I think my English led to a bit of a misunderstanding 😄

I mistook "his" as a typo for "this"